### ISSUE TITLE
Pathing

### ISSUE THREAD

**Issue by lanadelray** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3882]
Hello,

Pathing is one of the first things a new player will notice when joining the server. Pathing on archangel is pretty terrible so I can imagine that quite a few people have logged on and said holy crap this is ridiculous no thank you.

It makes tanking / DPSing troublesome, and I can't imagine it's effects in PvP (I don't partake). I've played on archangel a long time and it wasn't always like this. There was a change probably around a year ago that created these issues. 

My personal observations are that anytime there is any sort of cc on a mob (slow, stun, fear, etc.), it does something weird where it will run in place off somewhere in the distance but it won't actually be there. I have created a video and tried to recreate the issues I know of. I'll keep recording game play and posting here as it only gets more ridiculous as you go into dungeons / raids.

If you'd like to see what normal pathing looks like ... I mean just open retail wow or any retail wow video it hasn't changed really.

https://youtu.be/y6Vdh2G8z10

**Comment by Porshia** [comment]
Yeah, this is one of the major issues that needs to be addressed ASAP.

The improper pathing makes PvP a nightmare and also affects the quality of raiding.

So it would be nice if we get a developer to look into this and solve it, which will increase the quality of the server to a very greater extent 🙂

**Comment by Pawx** [comment]
It worked better till the illidan pathing fix a long time ago :/. Then CC`d mobs, especially if FEARed just go crazy for some reason.

**Comment by lanadelray** [comment]
more of the same!

https://youtu.be/KjAS6bvrtIk

**Comment by Porshia** [comment]
Hey Bytekiller. How's it going man? Any progression/update on this issue?

**Comment by ghost** [comment]
@Porshia some progress on the issue. I have found some issues that's causing/contributing to the problem. It's requiring code analysis to narrow it down further to make the necessary changes. May be able to speed up process, but it needs some discussion first before I can do that.

**Comment by Zerixx** [comment]
Fixed 12-04-16.

**Comment by Porshia** [comment]
Awesome!!
Fixed on Live? (I dont think so..) 

Or will it be implemented during weekly maintainence?

**Comment by Zerixx** [comment]
We'll see it go live in the coming days to week. This type of change can't go without rigorous testing. That said, it's looking decent on my end :)

**Comment by Porshia** [comment]
That sounds great. Hope everything goes fine and this one gets fixed on Live soon(like this friday!).
Great job @Zerixx and @ByteKiller  :)
